pkg/plugins/resources/bitbucket/client/main.go

```diff
@@ -52,9 +52,7 @@ type Spec struct {
 	Repository string `yaml:",omitempty" jsonschema:"required"`
 }
 
-type Client *scm.Client
```

Comment:
Removed the type alias so the [`b.client.Do`](https://github.com/drone/go-scm/blob/a515870a3931d01562ab61723891462134a3277f/scm/client.go#L157) method could be used.

Reply:
Thanks for the explanation